service/pipeline.ml

```diff
@@ -148,14 +153,15 @@ let v ~app () =
   let jobs = builds
              |> List.map (fun (variant, _build, job) ->
                  let+ x = job in
-                 (variant, Current.Analysis.job_id x)
+                 (variant, x)
                )
              |> Current.list_seq
   in
   let index =
     let+ commit = head
+    and+ analysis = job_id analysis
     and+ jobs = jobs in
-    Index.record ~commit jobs
+    Index.record ~commit @@ ("ANALYSIS", analysis) :: jobs
```

Comment:
Why so shouty here? Most of the other job names are lower case

Reply:
It's to make it appear first in the list. Otherwise, if it fails then it's not obvious why the first items in the list haven't started (and "alpine" comes before "analysis", so there are a lot of them).